README.md

```diff
@@ -421,6 +421,9 @@ A sample executable error response:
 These are all required fields for an error response. The code and message
 fields will be used by the library as part of the thrown exception.
 
+For successful responses, the `expiration_time` field is only required
+when an output file is specified in the credential configuration. 
+
 Response format fields summary:
   * `version`: The version of the JSON output. Currently only version 1 is supported.
   * `success`: The status of the response. When true, the response must contain the 3rd party token, 
```

Comment:
the summary for the success field here still says the response must contain expiration if successful, change to say that expiration time is optional